libs/core/src/schemas/events.schemas.ts

```diff
@@ -46,3 +46,25 @@ export const DiscordMessageCreated = z.object({
     z.literal('comment-create'),
   ]),
 });
+
+/**
+ * Zod schema for EvmEvent type defined in workers/evmChainEvents/types.ts
+ */
+export const ChainEventCreated = z.object({
```

Comment:
wondering if the typings can be improved with regex validation or something else? ...just to avoid downstream errors.

Reply:
I'm assuming you are talking about `parsedArgs`? Each event has different parsed args, I can potentially have a union but it can't be restrictive since paraedArgs is not an object that can be represented in zod completely.

Reply:
not a big deal, just if we can find it useful for downstream consumers

Reply:
https://github.com/hicommonwealth/commonwealth/issues/7444 I created another issue for looking into this.